Makefile

```diff
@@ -27,7 +27,7 @@ lint:
 	${FLAKE8} phy --exclude=phy/ext/*,default_settings.py --ignore=E226,E265,F811
 
 test: lint
-	py.test --cov-report term-missing --cov phy --ignore experimental
+	py.test --cov-report term-missing --cov phy --ignore experimental -s
```

Comment:
ah, sorry, I misread the man page, I thought -s disabled the output, whereas it actually disables the output __capturing__...!